source/manuals/programming-languages/html.html.md.erb

```diff
@@ -94,6 +69,31 @@ Examples include author information, related links and related content.
 
 `<footer role="contentinfo">` should be used for the footer of the site.
 
+### Region landmarks
+
+Region landmarks can be used when you need to create a new type of landmark.
+
+You can create region landmarks by using the `<section>` tag with an
+`aria-labelledby` attribute linking to the id of the heading that identifies
+it.
+
+Region landmarks should only be used where:
+
+- users will likely want to be able to navigate to the grouping easily and to
+  have it listed in a summary of the page
+- none of the existing HTML5 tags or ARIA landmark roles used for document
+  structure would correctly identify the grouping
+
+The use of region landmarks should be limited, as too many landmarks can dilute
+their usefulness in quickly navigating to important areas of a page. If you
+find there are many custom landmarks then the page’s structure should be
+re-examined.
+
+Note that use of `aria-labelledby` to assign an [accessible
+name](https://www.tpgi.com/what-is-an-accessible-name/) is required for region
+landmarks. Without an accessible name `<section>` tags are semantically the
+same as `<div>` tags.
```

Comment:
Technically it's the accessible name which is required here – you could also achieve that by using `aria-label`. Suggest rewording to avoid this being misinterpreted as a requirement to use `aria-labelledby` over `aria-label`.

This could also potentially be moved further up and merged with the second paragraph to avoid some repetition:

> Region landmarks can be used when you need to create a new type of landmark.
> 
> You can create region landmarks by using the `<section>` tag. The `<section>` must have an [accessible
name](https://www.tpgi.com/what-is-an-accessible-name/), which can be provided using the [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) or [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) attributes. A [`<section>` tag without an accessible name](https://w3c.github.io/html-aam/#el-section) is semantically the same as a `<div>` tag.

(I've added a few extra reference links as suggestions – not sure how useful they are)

Reply:
Should be addressed in https://github.com/alphagov/gds-way/pull/781/commits/77471320031391297a8775d31e9007280ebe0f2a.